apiserver/plane/middleware/logger.py

```
@@ -69,3 +71,41 @@ def __call__(self, request):
+                    body=(request_body.decode("utf-8") if request_body else None),
+                    response_body=(
+                        response.content.decode("utf-8") if response.content else None
+                    ),
+                    response_code=response.status_code,
+                    ip_address=get_client_ip(request=request),
+                    user_agent=request.META.get("HTTP_USER_AGENT", None),
+                )
+
+            except Exception as e:
+                api_logger.exception(e)
+                # If the token does not exist, you can decide whether to log this as an invalid attempt
```

Ruff (0.8.2)

Line too long (103 > 88)

(E501)